src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -640,6 +649,7 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
 	
 	func test_screenshot_maskOptionalForFederalStateBW() throws {
 		app.setLaunchArgument(LaunchArguments.healthCertificate.testCertificateRegistered, to: true)
```

Comment:
Do we need still need testCertificateRegistered? or we can just use firstAndSecondHealthCertificate and isMaskOptionalStateEnabled ?

Reply:
Good Catch! changed it to firstAndSecondHealthCertificate.